config.py

```diff
@@ -0,0 +1,2 @@
+WTF_CSRF_ENABLED = True
+SECRET_KEY = 'Cthulhuphant_can_cause_annihilation'

```

Comment:
that does not make sense, unless I am missing something — the server will not request the page before doing a webhook request :)